docs/Getting Started/Example Apps/index.md

```diff
@@ -12,4 +13,208 @@ The example apps implement soft buttons, template text and images, a main menu a
+!!!
+
+### Getting Started
+
+We assume that you have [SDL Core](https://github.com/smartdevicelink/sdl_core) (We recommend Ubuntu 16.04) and an [HMI](https://github.com/smartdevicelink/generic_hmi) set up prior to this point. Most people getting started with this tutorial will be using Sdl Core and our Generic HMI. If you don't want to set up a virtual machine for testing, we offer [Manticore](https://smartdevicelink.com/resources/manticore/), which is a free service that allows you to test your apps in the cloud.
+
+!!! NOTE
+Sdl Core and an HMI or Manticore are needed to run Hello Sdl and to ensure that it connects.
+!!!
+
+### Configuring Core 
+
+To let Sdl Core connect to your app, first you will have to know the IP address of the machine that is running the Hello Sdl app. If you don't know what it is, running ```ifconfig``` in the terminal will usually let you see it for the interface you are connected with to your network. 
+
+After getting the IP address, you will have to set App ID, App Websocket Endpoint, and App Nicknames in Sdl Core to let it know where your instance of Hello Sdl is running. 
+
+!!! NOTE
+The App Websocket Endpoint contains the IP Address and port as the following: `ws://<ip address>:<port>/`.
+!!!
+
+
+#### Manticore
+If you are using Manticore, the app information can be easily set in the settings tab:
+
+![Main Screen](assets/manticore1.png)
+
+!!! NOTE
+Manticore needs to access you machine's IP address to be able to start a websocket connection with your cloud app. If you are hosting the cloud app on your local machine, you may need to do extra setup to make your machine publicly accessible. The other solution is to setup Core and HMI on your machine instead of using Manticore so Core can access your local IP address.
+!!!
+
+#### Sdl Core and Generic HMI
+If you are using Sdl Core and Generic HMI, you will have to add a policy table entry as the following one to the existing entries:
+
+```JSON
+ "8678309": {
+     "keep_context": false,
+     "steal_focus": false,
+     "priority": "NONE",
+     "default_hmi": "NONE",
+     "groups": ["Base-4"],
+     "RequestType": [],
+     "RequestSubType": [],
+     "hybrid_app_preference": "CLOUD",
+     "endpoint": "ws://<ip address>:<port>",
+     "enabled": true,
+     "auth_token": "",
+     "cloud_transport_type": "WS",
+     "nicknames": ["Hello Sdl"]
+ }
+```
+
+ Fore more information about policy tables please visit [Plicy Tables Guides](https://smartdevicelink.com/en/guides/sdl-server/api-reference-documentation/policy-table/overview).
+
+!!! NOTE
+Don't forget to replace `ws://<ip address>:<port>` with your own IP address and app port. The port that is used in Hello Sdl App is `5432`. It can be changed to a different port by modifying the number in `Main.java` class.
+!!!
+
+Following this, you should see an application appears on HMI as in the following screenshot:
```

Comment:
```suggestion
If the policy table configuration is correct, you should see the app icon show up on the HMI like the following screenshot:
```

Reply:
Actually, the app icon will not show up until the SDL app connects at least once. This is because the app is given the url where the app icon is stored. The app then downloads the image and sends it to Core.